packages/@glimmer/interfaces/lib/compile/wire-format.d.ts

```diff
@@ -79,6 +79,8 @@ export const enum SexpOpcodes {
   GetFreeAsComponentOrHelperHead = 35,
   // a helper or `this` fallback `attr={{x}}`
   GetFreeAsHelperHeadOrThisFallback = 36,
+  // a helper or `this` fallback (deprecated) `@arg={{x}}`
+  GetFreeAsDeprecatedHelperHeadOrThisFallback = 99,
```

Comment:
Why 99?

Reply:
This is temporary and will be removed again soon, so seems somehow safer to use a number that wouldn’t be used to mean something else important anytime soon (though if the WF and compiler version doesn’t match you are going to have a bad time anyway)